### DISCUSSION TITLE
Cam Verification - Yi Dome Guard

### DISCUSSION THREAD

**Discussion by rpkilgore** [roleoroleo/yi-hack-Allwinner-v2#262] [Q&A] [answered]
Howdy, I'm back again... Just to verify looks like I picked another cam that is in beta.. lol just wanted to make sure though.

It's called a Yi Dome Guard, Serial starts with IFUSY36 and firmware ver is 9.0.05.33.
and here is the dump also.
[backup_DomeGuard.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/7180378/backup_DomeGuard.zip)

I think this is the one you have mentioned as being in beta, the r30gb_0.1.9.  Just wanted to make sure I'm using the correct hack and If it is the correct hack what part is in beta?  Any functionality I can verify for you?

**Update**
Ok, looks like I chose correctly.  So  far issues I've noticed:

1) Camera will reboot when you take a snapshot
2) looks like it's a bit glitchy when viewing the high res stream

Only other question I have is how do you disable the 1 min long every 1 min recordings in the Motion Events?

**Comment by roleoroleo** [comment, marked as answer]
Yes, this cam is a r30gb.
It's still in beta because is the only model that uses h265.

>     1. Camera will reboot when you take a snapshot
To fix the reboot try to enable swap space.

> Only other question I have is how do you disable the 1 min long every 1 min recordings in the Motion Events?
I don't know how to fix this issue.

**Reply by rpkilgore** [reply]
Thanks, that fixed it right up.
